blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
+ title:
+ layout: post
+ created_at: Fri Oct 12 2016
+ permalink: blog/2016-09-30-Hackbrighter-End-of-Summer
```

Comment:
could you please make the date (`created_at`), permalink and filename consistent? Ideally both file and permalink should have the same name, and the date should be the same everywhere